index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formulário</title>
+</head>
+<body>
+    <h1>Time de Valorant</h1>
+    <p>Formulario para montar um time de valorant</p>
+    <form action="mailto:alefstarkin@gmail.com">
+        <div>
+            <label>Nome:</label><input type="text" placeholder="Digite seu nome" required>
```

Comment:
Os elementos `label` tem como função semantica identificar o proprósito de um input, e uma forma que o HTML possibilita de fazer essa identificação é através da propriedade `for` do `label`, que recebe o nome de um `id` para usar de referencia.

```suggestion
            <label for="nome">Nome:</label><input id="nome" type="text" placeholder="Digite seu nome" required>
```

vale lembrar que, sem a utilização do `for` no `label` estamos abrindo mão do seu principal recurso.